mlos_bench/mlos_bench/tests/config/services/remote/mock/mock_vm_service.jsonc

```diff
@@ -0,0 +1,6 @@
+{
+    "class": "mlos_bench.tests.services.remote.mock.mock_vm_service.MockVMService",
+    "config": {
+        "vmName": "test-vm"
```

Comment:
One other issue with this for the future: we're going to need some reworks to support multiple different VmEnvs that target different VMs I think.